### ISSUE TITLE
Add a timer for expboost / moneyboost etc.

### ISSUE THREAD

**Issue by RSLuke** [Zrips/Jobs#491]
**Description of new idea or improvement on existing one**

It's simple, the goal would be to have the ability to add a timer after /jobs expboost rate [TIME]

Example :
/jobs expboost all 1d14h22m -> Would boost all of the jobs for 1 day, 14 hours and 22 minuts. This way it could really enhance mechanics around this kind of boosting. You could probably take libraries from CMI to get the time things.

**Comment by ArathiaMC** [comment]
I agree. This would be good and eliminate the need for external booster plugins to effectively apply job booster events and limited time boosts.

**Comment by montlikadani** [comment]
This will again be a memory leak to the server and again we will receive lots of reports.

**Comment by AntoineHrn** [comment]
@montlikadani 
But how did Zrips managed to work with of the things related to delays and time stuff on CMI ?

@ArathiaMC what plugin do you use to do achieve this ?

**Comment by montlikadani** [comment]
The same problem will be, I have seen many reports in the CMI that a lot of memory leaks, but usually when we do the tasks, when that task is done, we stop it, thus preventing the memory leak.

**Comment by ArathiaMC** [comment]
@AntoineHrn 
I am using a custom one but there should be something like this in the plugin itself.

**Comment by AntoineHrn** [comment]
Well we'd probably all love if you'd find a solution for this to happen :D

**Comment by montlikadani** [comment]
https://github.com/Zrips/Jobs/commit/f649b004cd27037c78a5f5d7c810d2f97fc15cd9

**Comment by montlikadani** [comment]
Just with a little fixes: https://github.com/Zrips/Jobs/commit/4115bf727a9c54933d1d50e1350272d7b531f001
